cbw_api_toolbox/cbw_api.py

```diff
@@ -79,6 +79,17 @@ def server(self, server_id):
 
         return CBWParser().parse_response(CBWServer, response)
 
+    def add_group_server(self, server_id, groups_name):
+        """PATCH request to /api/v2/servers for server groups"""
+        if server_id:
+            response = self._request("PATCH", [ROUTE_SERVERS, server_id], {'groups': groups_name})
+
+            print("add group")
+
+            return self.verif_response(response)
+        print("No server id")
```

Comment:
Add new line between `return self.verif_response(response)` and ` print("No server id")`